src/inmanta/data/dataview.py

```diff
@@ -1313,7 +1321,8 @@ def construct_dtos(self, records: Sequence[Record]) -> Sequence[dict[str, str]]:
                 discovered_resource_id=res["discovered_resource_id"],
                 values=json.loads(res["values"]),
                 managed_resource_uri=f"/api/v2/resource/{res['discovered_resource_id']}" if res["managed"] else None,
-            ).model_dump()
+                discovery_resource_id=res["discovery_resource_id"],
+            ).model_dump(exclude={"discovery_resource_id"})
```

Comment:
Shouldn't we make this exclude part of the config of the pydantic class, instead of setting this parameter in every location where we serialize the object.

Reply:
I have a suspicion that the regular GET endpoint to retrieve a specific discovered resource is returning this field, while the batch endpoint isn't. Is that correct? If this is the case, we need to improve the test coverage.